api/src/main/java/org/openmrs/module/registrationapp/model/RegisterPersonRelationshipWidget.java

```diff
@@ -0,0 +1,79 @@
+package org.openmrs.module.registrationapp.model;
+
+import org.codehaus.jackson.annotate.JsonProperty;
+
+public class RegisterPersonRelationshipWidget extends Widget{
+
+    private Config config = new Config();
+
+    public RegisterPersonRelationshipWidget() {
+        super("registrationapp", "field/registerPersonRelationship");
+    }
+
+    public Config getConfig() {
+        if (config == null) {
+            config = new Config();
+        }
+        return config;
+    }
+    public void setConfig(Config config) {
+        this.config = config;
+    }
+
+    public static class Config {
```

Comment:
See my comments on the pihcore PR.  Basically, let's make this a top-level class named `PersonRelationshipConfig` and also add the `label` property you have on the config in the other PR.

Reply:
@mseaton  I would think it makes sense to keep this like it is, because it follows the existing pattern used for the other widgets?  Or am I missing something?

Reply:
Ok, fine if that's what we've done...